src/globalInstance.ts

```diff
@@ -15,10 +16,19 @@ export function getGlobalInstance(): Monitor {
 }
 
 export const monitored: Monitor['monitored'] = (...args) => getGlobalInstance().monitored(...args);
+
 /**
  * @deprecated since version 2.0
  */
 export const getStatsdClient: Monitor['getStatsdClient'] = (...args) => getGlobalInstance().getStatsdClient(...args);
 export const increment: Monitor['increment'] = (...args) => getGlobalInstance().increment(...args);
 export const gauge: Monitor['gauge'] = (...args) => getGlobalInstance().gauge(...args);
 export const timing: Monitor['timing'] = (...args) => getGlobalInstance().timing(...args);
+
+export const monitorMethod = <T>(prefix?: string, options: MonitoredOptions<T> = {}) => {
+    return function (_target: Object, propertyKey: string, descriptor: PropertyDescriptor) {
+        const originalMethod = descriptor.value;
+        const metricName = `${prefix}${prefix ? '.' : ''}${propertyKey}`;
+        descriptor.value = monitored(metricName, originalMethod, options);
```

Comment:
```suggestion
        descriptor.value = (...args) => monitored(metricName, () => originalMethod(...args), options);
```

monitored calls the method, you need `descriptor.value` to be a method